tpl/js-templates.php

```diff
@@ -5,66 +5,70 @@
 
 <script type="text/template" id="siteorigin-panels-builder">
 
-	<div class="so-builder-toolbar">
+	<div class="siteorigin-panels-builder">
 
-		<a href="#" class="so-tool-button so-widget-add">
-			<span class="so-panels-icon so-panels-icon-plus"></span>
-			<span class="so-button-text"><?php esc_attr_e('Add Widget', 'siteorigin-panels') ?></span>
-		</a>
+		<div class="so-builder-toolbar">
 
-		<a href="#" class="so-tool-button so-row-add">
-			<span class="so-panels-icon so-panels-icon-columns"></span>
-			<span class="so-button-text"><?php esc_attr_e('Add Row', 'siteorigin-panels') ?></span>
-		</a>
-
-		<a href="#" class="so-tool-button so-prebuilt-add">
-			<span class="so-panels-icon so-panels-icon-cubes"></span>
-			<span class="so-button-text"><?php esc_attr_e('Prebuilt', 'siteorigin-panels') ?></span>
-		</a>
-
-		<?php if( !empty($post) ) : ?>
+			<a href="#" class="so-tool-button so-widget-add">
+				<span class="so-panels-icon so-panels-icon-plus"></span>
+				<span class="so-button-text"><?php esc_attr_e('Add Widget', 'siteorigin-panels') ?></span>
```

Comment:
I realize it's not something you changed, but just pointing it out while I'm here. It's a small thing, but couldn't all these be using `_e(...`? (and similarly for 'Add Row', annd 'Prebuilt' below)